src/parser/index.js

```diff
@@ -168,7 +168,7 @@ walkDir(input, (map, filename) => {
   parsedMap.connections = map.connections.map(conn => parseConn(conn, nodesLookup));
 
   // Find out the path for the output file.
-  const inputBasePath = `${path.resolve(path.join(__dirname, '../../'), input)}/`;
+  const inputBasePath = path.resolve(path.join(__dirname, '../../'), input);
```

Comment:
I checked and it works, I changed `path.join(__dirname, '../../')` to `'./'` cause I exported that script, so I can use it on the search engine, and it wasn't working properly with `__dirname`.